generator/funcs.go

```diff
@@ -189,7 +190,31 @@ func (a *Generator) colnames(fields []*internal.Field, ignoreNames ...interface{
 		str = str + a.colname(f.Col)
 		i++
 	}
+	return str
+}
+
+// colnames creates a list of the column names found in fields, excluding any
```

Comment:
Please fix the comments.

Reply:
Thank you for your comment.
I fixed.
Please check it.